modules/ding_adgangsplatformen/includes/ding_adgangsplatformen.admin.inc

```diff
@@ -19,6 +19,29 @@ function ding_adgangsplatformen_admin_settings_form($form, &$form_state) {
 
   $default = ding_adgangsplatformen_get_configuration();
 
+  $form['ding_adgangsplatformen_settings']['auth_client'] = array(
+    '#type' => 'fieldset',
+    '#collapsible' => FALSE,
+    '#title' => t('Auth client'),
+    '#description' => t('Due to technical issues, DDB needs to authenticate users using a client with elevated privileges, before dropping down to a client with lesser privileges for general use.'),
+  );
+
+  $form['ding_adgangsplatformen_settings']['auth_client']['authClientId'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Client ID'),
+    '#size' => 35,
+    '#description' => t('The client id should be obtained from the authentication provider - DBC.'),
+    '#default_value' => $default['auth_client']['authClientId'],
```

Comment:
I have some concerns with this approach.

As I understand it sites are supposed to use these fields for the site-specific privileged client/id. Without this PR they are currently held within the `clientId`/`clientSecret`. With this change each site would have to move it the values to `['auth_client']['authClientId']`/`['auth_client']['authClientSecret']` and back again when the double token hack is no longer needed.

I think it would be preferable to keep the current structure and keep the extra client id/secret separate. When the hack is no longer needed and the current client id/secret looses their privilege we can just remove our changes. 

I wonder if it is a good decision to provide any UI and rely on the current nested structure for this? All sites should use the same secret/id and these should not be changed. I would be OK with using "flat" variables with can be defined using `settings.php`.